packages/vue/src/components/molecules/SfComponentSelect/SfComponentSelect.stories.js

```diff
@@ -137,6 +137,17 @@ NoChevron.args = {
   classes: "sf-component-select--no-chevron",
 };
 
+export const LongOptionsList = Template.bind({});
+LongOptionsList.args = {
+  ...Common.args,
+  options: [
```

Comment:
I think you can just change default size to 3 instead of adding more items to options and it will be enogh to see the effect. Wdyt?

Reply:
@AdamPawlinski done :)